store/auth.js

```diff
@@ -18,61 +21,62 @@ export const mutations = {
   },
   resetToken(store) {
     store.token = null;
-  },
+  }
 };
 
 export const actions = {
   async refreshUserDetails(
     { commit, dispatch },
-    { token = cookies.get("token"), base = "" } = {}
+    { token = cookies.get("token") } = {}
   ) {
-    let headers = {};
+    var me;
     if (process.server) {
-      headers = { cookie: `token=${token}` };
-    }
-    var res;
+      me = await module.getSession(token);
+    } else {
+      var res;
+
+      try {
+        res = await fetch(`/auth/me`, {
+          credentials: "include"
+        });
```

Comment:
how did you get this to work?

Reply:
I stopped fetching the same server and used a plugin to connect to auth module (only on server). fetch is only used on client